wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/apriltagsvision/Robot.java

```diff
@@ -39,14 +39,14 @@ public void robotInit() {
 
   void apriltagVisionThreadProc() {
     var detector = new AprilTagDetector();
-    // look for tag16h5, don't correct any error bits
-    detector.addFamily("tag16h5", 0);
+    // look for tag136h11, correct 7 error bits
```

Comment:
```suggestion
    // look for tag36h11, correct 7 error bits
```

There's a typo here...